packages/backend/README.md

```diff
@@ -74,7 +75,7 @@ environment variables. One variable per line `KEY=value`.
 - `DISCOVERY_BLOCK_NUMBER` (Optional) - Override the block number used during local discovery
 - `UPDATE_MONITOR_RUN_ON_START` (default `true`) - Prevent UpdateMonitor from running on start when developing locally
 
-#### .env boilerplate:
+#### .env boilerplate
```

Comment:
The boilerplate `.env` needs to have it's entry for `ALCHEMY_API_KEY=` replaced with `ETHEREUM_ALCHEMY_API_KEY=` and also add the `ARBITRUM_ALCHEMY_API_KEY=` to the optional.

Reply:
https://github.com/l2beat/l2beat/pull/1645/commits/8edfed7853adab335a3d357221d9749e545ac0aa
https://github.com/l2beat/l2beat/pull/1645/commits/04589bc72f1450a84a1efb299d7fb21792794eb7